cmd/wire/main.go

```diff
@@ -115,6 +116,7 @@ func (*genCmd) Usage() string {
 }
 func (cmd *genCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&cmd.headerFile, "header_file", "", "path to file to insert as a header in wire_gen.go")
+	f.StringVar(&cmd.prefixFileName, "prefix", "", "prefix to the output files concat with 'wire_gen.go' suffix.")
```

Comment:
Nit: let's call this flag `output_file_prefix` to be more specific. And please reword the help text to "string to prepend to output file names."

Thanks!

Reply:
I just did as you ask @vangent. You were right about the "prefix" name, is more understandable  `output_file_prefix`